### ISSUE TITLE
[Bug][a11y]: Talkback doesn't read the Desktop site toggle state

### ISSUE THREAD

**Issue by SoftVision-LorandJanos** [mozilla-mobile/fenix#25929]
### Steps to reproduce

1. Turn TalkBack (or the available screen reader) on.
2. Go to 3 dot menu.
3. Select the desktop site toggle
4. Double tap on the toggle to change its state.
5. Observe the behavior.

### Expected behaviour

Talkback reads the state of the desktop site toggle.

### Actual behaviour

Talkback doesn't read the state of the desktop site toggle.

### Device name

One Plus 9 Pro

### Android version

Android 12

### Firefox release type

Firefox Beta

### Firefox version

103.0.0-beta2

### Additional information

The same result with desktop site toggle turned on or off.

https://user-images.githubusercontent.com/48829350/177788188-27d95146-d487-4428-9f90-011946e8e069.mp4

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-20941)

**Comment by SoftVision-LorandJanos** [comment]
This issue is no longer reproducible, I will close it as not planned.
